dotCMS/src/main/java/com/dotmarketing/beans/Identifier.java

```diff
@@ -252,6 +254,15 @@ public String getCategoryId() {
 	   return getInode();
 	}
 	
+	   /**
+     * Returns the Identifier URI, 
+     * based on its parent path (folder) and asset name.
+     * Appends a forward slash for folder ids.
+     * Useful for retrieving file/page URLs
+     * 
+     * @return String
+     * @see getURI
+     */
 	public String getPath(){
 	    if(getAssetName().equals("system folder") && getParentPath().equals("/System folder"))
```

Comment:
you might want to use constants and the constant in the left side, such as

SYSTEM_FOLDER.equals(getAssetName())